test/parallel/test-repl-top-level-await.js

```diff
@@ -173,6 +173,10 @@ async function ordinaryTests() {
       '3',
       'undefined',
     ]],
+    ['const k = await Promise.resolve(123)'],
+    ['k', '123'],
+    ['k = await Promise.resolve(234)', '234'],
+    ['k', '234'],
```

Comment:
And the drive the point home:
```suggestion
    ['k', '234'],
    ['const k = await Promise.resolve(345)'],
    ['k', '345'],
```

Reply:
line 180 causes error like 'Uncaught SyntaxError: Identifier 'k' has already been declared'.
When I tested,
```
// This is allowed
const k = await Promise.resolve(123)
k = await Promise.resolve(234)

// This is not allowed
const k = await Promise.resolve(123)
const k = await Promise.resolve(234)
```